community/bolt/src/test/java/org/neo4j/bolt/v1/runtime/internal/concurrent/ThreadedSessionsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.bolt.v1.runtime.internal.concurrent;
+
+
+import org.junit.Test;
+
+import org.neo4j.bolt.v1.runtime.Session;
+import org.neo4j.bolt.v1.runtime.Sessions;
+import org.neo4j.kernel.impl.logging.NullLogService;
+import org.neo4j.kernel.impl.util.JobScheduler;
+import org.neo4j.kernel.monitoring.Monitors;
+
+import static org.mockito.Mockito.mock;
+
+public class ThreadedSessionsTest
+{
+    @Test
+    public void shouldMonitorSessions() throws Throwable
+    {
+        // Given
+        Monitors monitors = new Monitors();
+        ThreadedSessions sessions = new ThreadedSessions( mock( Sessions.class ), mock( JobScheduler.class ),
+                NullLogService.getInstance() );
+
+        Session session = sessions.newSession();
+
+        // When
+        session.run( "hello", null, null, null );
```

Comment:
@jakewins  I don't think there's much point in this test. There is no monitoring inside this class...